src/Hook/ActionObjectShopUpdateAfter.php

```diff
@@ -31,50 +31,54 @@ class ActionObjectShopUpdateAfter extends Hook
      * @param array $params
      *
      * @return bool
-     *
-     * @throws Exception
      */
     public function execute(array $params = [])
     {
-        /** @var Link $link */
-        $link = $this->module->getService(Link::class);
+        $this->updateUserShop($params['object']);
```

Comment:
Maybe reinstanciate a Shop object as before to be consistent and not depending on prestashop for it (difference between version, inconsistency, ...)

Reply:
I think that's not necessary as long as the associated feature test is passing.